service/update.check/pom.xml

```diff
@@ -88,6 +81,8 @@
                 <configuration>
                     <additionalClasspathElements>
                         <additionalClasspathElement>${basedir}/src/test/resources/gateway.server-5.0.0.8.jar</additionalClasspathElement>
+                        <argLine>${surefireArgLine}</argLine>
+                        <skipTests>${skipTests}</skipTests>
                     </additionalClasspathElements>
                 </configuration>
             </plugin>
```

Comment:
The plugin version probably is not setup in kaazing/community which is where it would inherit the version from.